pages/releases.js

```diff
@@ -40,8 +40,14 @@ const Releases = ({ releases, sidebarPages }) => (
   </DocsLayout>
 );
 
-Releases.getInitialProps = async () => {
+Releases.getInitialProps = async ({ res }) => {
   const releases = await getReleases();
+
+  if (res) {
+    // Revalidate this data every 30 seconds
+    res.setHeader('cache-control', 's-maxage=30,stale-while-revalidate');
```

Comment:
I think we could totally increase this to a higher value. We don’t really need this page to be _too up-to-date_ 😅

Reply:
Since the stale-while-revalidate header is present you'll never receive a CDN MISS, even after 30 seconds.

Reply:
![](https://media.giphy.com/media/oYtVHSxngR3lC/giphy.gif)